Ginger/Ginger/ALM/ALMIntegration.cs

```diff
@@ -576,5 +592,25 @@ public eUserMsgKeys GetDownloadPossibleValuesMessage()
         {
             return AlmRepo.GetDownloadPossibleValuesMessage();
         }
+
+        public List<string> GetTestLabExplorer(string path)
+        {
+            return AlmRepo.GetTestLabExplorer(path);
+        }
+
+        public List<QCTestSetSummary> GetTestSetExplorer(string path)
+        {
+            return AlmRepo.GetTestSetExplorer(path);
+        }
+
+        public QCTestSetSummary GetTSRunStatus(QCTestSetSummary tsItem)
+        {
+            return AlmRepo.GetTSRunStatus(tsItem);
+        }
+
+        public List<string> GetTestPlanExplorer(string path)
+        {
+            return AlmRepo.GetTestPlanExplorer(path);
+        }
     }
```

Comment:
Also here there are those "QCTestSetSummary" objects types which seems specific for QC but we are on ALMIntegration which supposed to be generic

Reply:
You asked to do the get test set and test plan explorer generic in the QCTestLabExplorerPage and QCTestPlanExplorerPage. In order to do it i need to declare it in the ALMRepository level.